test/system/500-networking.bats

```diff
@@ -106,7 +105,9 @@ load helpers
         # remote IP is not 127.0.0.1 (podman PR #9052).
         # We could get more parseable output by using $NCAT_REMOTE_ADDR,
         # but busybox nc doesn't support that.
-        run_podman run -d --userns=keep-id $network_arg -p 127.0.0.1:$myport:$myport \
+        userns="--userns=keep-id"
+        is_rootless || userns="--uidmap=0:1111111:65536 --gidmap=0:1111111:65536"
```

Comment:
DAE find the `||` form hard to read in this context? It takes my brain an extra second or two to interpret it. I know it takes up lines, but would anyone object to an `if/else`?

Reply:
I am fine with that, but the `is_rootless ||` is used in multiple places, so I went with that.